library/Rules/CreditCardVisa.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+class CreditCardVisa extends CreditCard
+{
+
+    public function validate($input)
+    {
+        $input = preg_replace('([^0-9])', '', $input);
```

Comment:
a few lines of code here are actually the same in the method `validate` of the class `CreditCard`.

what do you think about:
```php
return parent::validate($input) && $this->verifyIfCardIsVisa($input);
```
?